ios/Classes/NativeBridge/Handlers/RegistrationHandler.swift

```diff
@@ -143,29 +138,37 @@ extension RegistrationHandler {
     }
 }
 
-extension RegistrationHandler: RegistrationDelegate {
+class RegistrationDelegateImpl: RegistrationDelegate {
+    private let completion: ((Result<OWRegistrationResponse, FlutterError>) -> Void)
+    private let registrationHandler: RegistrationHandler
+
+    init(_ completion: @escaping (Result<OWRegistrationResponse, FlutterError>) -> Void, _ registrationHandler: RegistrationHandler) {
```

Comment:
please place `completion` at the end of params list and don't put `_` - it should be called like this ⬇️ anyway 
```
RegistrationDelegateImpl(registrationHandler: registrationHandler) { result in
...
}
```
and even if not in general params in Swift should be labeled, sometimes with contradiction style but still not sth like `RegistrationDelegateImpl(completion, handler, otherParam, etc)`  in this case 
RegistrationDelegateImpl(registrationHandler: handler, completion: completion) is fine.